nav2_bringup/launch/bringup_multirobot_launch.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2023 LG Electronics.
```

Comment:
We should rename this launch file to be in line with the other launch files

How's `multi_cloned_tb3_simulation_launch.py` and rename the existing one as `multi_unique_tb3_simulation_launch.py`

Reply:
Well, I think actually the role and purpose of each scripts are almost same. 
I mean `multi_tb3_simulation_launch.py` and `bringup_multirobot_launch.py`.

So, how about just replace existing `multi_tb3_simulation_launch.py` with `bringup_multirobot_launch.py` 
and change filename `bringup_multirobot_launch.py` -> `multi_tb3_simulation_launch.py``.

As you can see, the only difference is the parameters of namespace + pose.

Reply:
the prefix `bringup_` means something in particular if you look at the launch files that currently exist. That is bringing up the abstract Nav2 stack + localization / slam method of choice. The role of the `<robot name>_` prefix is to showcase instances of bringups with the specifics for particular robot platforms - involving their specific configurations, simulations, etc.

So we should keep the pattern of starting with the same prefix as we have currently - or potentially change to something else - but not reuse `bringup_`. These names should be specific to the (1) robot and (2) setup (e.g. multirobot), but abstracted. 

I'm not married to `multi_cloned_tb3_` and `multi_unique_tb3_` but something that denotes that its about the TB3 and that its about multiple robots with differences in their configurations is key. We should not remove any existing files, but we could rename the `N` TB3's with unique configuration files to be in the same naming convention as the `N` TB3's with the same configurations as your PR introduces. Its not a replacement, its an augmentation. There are still real uses of having multiple robots with different configurations spawned.

Reply:
I totally agree in terms of the meaning of `bringup_` and `bringup_` prefix is not appropriate for the script what I created.


As you commented, I need to denote on filename that this script shall launch the `TB3`.

I prefer to use just `multi_tb3_` as prefix but `multi_cloned_tb3_` is also OK to me. 

But does `multi_unique_tb3_` naming OK to previous user who used `multi_tb3_simulation_launch.py` usually?

_If you think and confirm it doen't matter, I will rename the file name what you suggested._

I've seen that the simulation script for `TB4` is preparing by another PR.
Maybe, it just my assumption, it needs to be modified for general case. For example, just parameterize `tb3_simulation_launch.py(?)`.
(Of course, later! and by separated PR as s future work)

Reply:
What are your suggested names for the 2 multi-robot launch files (yours + the existing one)? We need a name for both that are analogous to each other, I don't really care that much as long as they're appropriately descriptive + as concise as possible.

The TB4 is a topic for another time, but we would have tb4 files once that's available and deprecate the tb3 files over a couple of distributions.

Reply:
> What are your suggested names for the 2 multi-robot launch files (yours + the existing one)? We need a name for both that are analogous to each other, I don't really care that much as long as they're appropriately descriptive + as concise as possible.
> 

Well, I also prefer to use simple one but intutivie one (could be kind of tradeoff).

Actually, my suggestion is 

`bringup_multirobot_launch.py` -> `multi_tb3_simulation_launch.py`
`multi_tb3_simulation_launch.py` -> `multi_tb3_simulation_launch.{something denote previous one or parameters are already defined in code}.py`

for exmaple,
 `multi_tb3_simulation_launch.fixed.py`
 `multi_tb3_simulation_launch.unique.py` 
 `multi_tb3_simulation_launch.two-robots.py` -> cause there is only robot1 and robot2



> The TB4 is a topic for another time, but we would have tb4 files once that's available and deprecate the tb3 files over a couple of distributions.
Yeha, right. Over coubple of distributions :)

Reply:
The `_launch.py` suffix is set for us by the ROS best practices. Surely it would work, but we want to make sure Nav2 is a 'best practices' repository. 

I think `unique_multi_tb3_simulation_launch.py` and `cloned_multi_tb3_simulation_launch.py` is the best. Both unique/cloned have the same number of letters so they'll appear analog in the filesystem + relatively short. The only other way I could see it being is `homo` or `hetero` for homogeneous and heterogeneous multirobot. I don't think that looks as good and might spawn some critiques and/or jokes I'd rather avoid.

Reply:
Okey. Understood about `_launch.py` suffix. I prefer to follow the best practics. :)

Then, let's go with `unique_multi_tb3_simulation_launch.py` and `cloned_multi_tb3_simulation_launch.py` once.

We could be modified later if there is some cirituques.